dialect/sql/driver.go

```diff
@@ -160,5 +160,9 @@ type NullScanner struct {
 // Scan implements the Scanner interface.
 func (n *NullScanner) Scan(value interface{}) error {
 	n.Valid = value != nil
-	return n.S.Scan(value)
+	if n.Valid {
+		return n.S.Scan(value)
+	} else {
+		return nil
+	}
```

Comment:
```suggestion
	if n.Valid {
		return n.S.Scan(value)
	}
	return nil
```

Looks good, except for a minor style comment.